README.md

```diff
@@ -234,7 +234,7 @@ To overrides endpoint's settings add this snippet tou your config:
 ```yaml
 overrides:
   endpoints:
-    /onboarding/info:              # This is the endpoint in the OpenApi spec
+    GET /onboarding/info:              # This is the endpoint in the OpenApi spec
```

Comment:
This change seems to introduce a breaking behavior, please, consider bumping the major version accordingly.